src/functionalTest/java/uk/gov/hmcts/reform/professionalapi/RetrievePaymentAccountTest.java

```diff
@@ -39,6 +40,24 @@
     private String email = generateRandomEmail();
     private String lastName = "someLastName";
     private String firstName = "someName";
+    String orgIdentifierResponse;
+
+    @Before
+    public void setUp() {
+        Set<String> paymentAccounts = new HashSet<>();

```

Comment:
Remove this from @Before since it will be each time.

Reply:
Addressed here: https://github.com/hmcts/rd-professional-api/pull/835/commits/3754a1661c7e08d3e4d15304736ed52e7f38e123